joblog.go

```diff
@@ -13,13 +13,17 @@ import (
 type JobLog struct {
 	url         string
 	serviceName string
+	jobName     string
+	actionName  string
 	jobId       int64
 	Disable     bool
 	Err         error
 }
 
 type jobStartDto struct {
-	Servcie string      `json:"service"`
+	Service string      `json:"service"`
+	Job     string      `json:"job"`
+	Action  string      `json:"action"`
 	Param   interface{} `json:"param"`
```

Comment:
param is a struct

jobLog := joblog.New(url,
		"test", map[string]interface{}{"jobName":"111","actionName":"222","log": "this is test"})